src/nyc_trees/apps/event/helpers.py

```diff
@@ -2,17 +2,21 @@
 
 
 def user_is_rsvped_for_event(user, event):
-    # The filter uses user.id so that the user argument can be either
-    # a real User or the anonymous user (not logged in). The anonymous
-    # user is a SimpleLazyObject, which you cannot use in a filter.
-    return EventRegistration.objects\
```

Comment:
This comment and the fix it references (using `user.id` instead of just `user` fixed a bug we encountered several times).  I would like us to keep it in place.